backend/src/sync.js

```diff
@@ -91,6 +91,40 @@ async function saveBlocks(blocksInfo) {
                     });
                   })
                 ),
+                models.AccessKey.bulkCreate(
+                  blockInfo.transactions
+                    .filter(tx =>
```

Comment:
This filter looks redundant to me.

Reply:
we have to have this since we need tx.receiverId as accountId and map the transactions directly instead of transactions.actions mapping. That is why filter all the transactions with "AddKey" is necessary. @frol

Reply:
Sorry, I don't follow what you are saying.

The function passed into `filterMap` will just return an empty list if there are no AddKey actions, and that will be just fine.